src/protocol/transactions.rs

```diff
@@ -513,16 +528,48 @@ impl Fee {
         self.fee.ceil() as u64
     }
 
+    #[cfg(test)]
```

Comment:
🔧 You can get rid of `split`, it's only used in tests to test itself.

Reply:
done